packages/components/src/components/table/Table.tsx

```diff
@@ -0,0 +1,169 @@
+import * as React from 'react';
+import { Checkbox, Icon, Dropdown } from '..';
+
+export type TableProps = {
+  items: any[];
+  header: any[];
+  checkbox?: boolean;
+  className?: string;
+  rowsPerPage?: number;
+  showSorting?: boolean;
+  showPagination?: boolean;
+};
+
+export const Table: React.FC<TableProps> = ({
+  items,
+  header,
+  checkbox,
+  rowsPerPage,
+  className,
+  showSorting,
+  showPagination,
+}: TableProps) => {
+  const [data, setData] = React.useState(items);
+
+  const pageNumber = 1;
+  const totalPage = 20;
+
+  function hideOrShowCheckbox() {
+    if (checkbox) {
+      return <Checkbox label="" name="simple-checkbox" value="checkbox-1" />;
+    }
+  }
+
+  function hideOrShowSorting(value) {
+    console.log(value);
+    if (showSorting) {
+      return showIcon(value.key);
+    }
+  }
+
+  function hideOrShowPagination() {
```

Comment:
as it is a method that produce some JSX, by convention we put `"render"` in the name
... for example here it could be named `renderPagination()`